app/konnect/service-catalog/integrations/index.md

```diff
@@ -6,13 +6,17 @@ beta: true
 ---
 
 Integrations are applications - either {{site.konnect_short_name}}-internal applications or external 
-third-party applications - that you can enable or disable within Service Catalog.
 
 When enabled, an integration gives you and your teams the ability to:
 * Discover new resources
 * Create new services based on existing resources
+* View key events associated with the integration directly in the event tab in {{site.konnect_short_name}}. 
 
 ## Integrations
 
 {% include_cached service-catalog-integrations.html %}
 
+
+## Events
+
+Eventing allows you to view key events associated with your {{site.service_catalog_name}} integrations directly from the **Events** tab in the {{site.konnect_short_name}} UI. From here you can search or filter through events by keywords in the event title and filter by time frame.
```

Comment:
```suggestion
```
This doesn't belong here. As this page grows, it will have many more tiles. This info will get obscured, and will seem even more out of place. Can you please move it to the service catalog index page?

Reply:
Will do in post